data-import/src/main/resources/folijet/data-import/features/data-import-orders.feature

```diff
@@ -550,7 +552,6 @@ Feature: Test Data-Import holdings records
     And match response.orderFormat == "P/E Mix"
     And match response.purchaseOrderId == secondRecordOrderId
 
-  @Ignore
```

Comment:
as far as I remember this test has been ignored due to some issue related to "mix" open order creation. Does this test work now?

Reply:
Yes, it was working in my testing